### ISSUE TITLE
implement non-LBM data support

### ISSUE THREAD

**Issue by alihaydaroglu** [alihaydaroglu/suite3d#40]
@landoskape is helping implement this for his data

main idea: 
- new branch to the function `load_and_stitch_tifs` to load non-LBM non-mesosocope tifs in the correct format
- add parameter 'lbm' (defaults to True) which will be passed to this function to decide whether to follow the generic or LBM branch
- add parameters to specify which channel (green/red) we care about

additional considerations:
- probably should keep the plane-to-plane xy alignment
- GPU registration needs to be adapted to skip the fusing step
- should check `'subtract_crosstalk'` parameter is set to false if LBM = False

**Comment by landoskape** [comment]
## Parameterization for standard 2P data
- There's a parameter called ``n_ch_tif`` which indicates the number of recorded planes in the recording. This is used for both LBM IO and standard 2P IO. 
- However, there are two parameters for which planes to analyze that are probably serving the same purpose:
   - There's a parameter for ``planes`` that indicates which planes to analyze for LBM data. 
   - There's another parameter called ``multiplane_2p_use_planes`` used around [here](https://github.com/alihaydaroglu/suite3d/blob/dev/suite3d/io/s3dio.py#L181) that indexes the planes to analyze for suite2p data. 

Can we just consolidate and use the same parameter for each context?

**Comment by landoskape** [comment]
## Incomplete volumes per tiff
The solution in #81 isn't going to work because of batched loading. Oops. 

Plan that will work: 
Whenever we are starting a job with standard 2P data, we'll extract the number of frames in each tif file, and figure out where volumes are incomplete. The result will be a dictionary in params like this:

``dict[tif_path, num_pages_at_end_that_belong_to_the_next_tiff]``

This way, whenever the ``_load_scanimage_tifs`` receives a path, it can look up how many pages it needs from the previous tif (if it isn't the first) and how many pages to ignore at the end. ``self.job.params["tifs"]`` has this info.

This way we can specifically load the pages we need and ignore the pages we don't need. 


## Tiff caching? 
Due to the slow speed of tiff loading, it could be nice to cache tiff reads in the job directory as temporary files... thoughts on this? There could be a mechanism to clear temporary tiff reads after the last step in the suite3d pipeline? Thoughts?

**Comment by alihaydaroglu** [comment]
Yeah, I think that makes sense! I am not sure about the tiff caching - it would only really be useful for this step, not registration? During registration the tiff loading is pipelined so I don't actually think it will cause a speedup, but I'm not sure. In my case, my tiffs live on the server, and I don't wait at all for I/O while doing registration since it happens in the background while processing the previous tiff
________________________________
